Adding Wave changelogs from backlog

christopher-hakkaart opened the pull request:
- Add changelogs back to v1.8.0

justinegeffen commented on changelog/wave/v1.10.2.mdx:
```diff
@@ -0,0 +1,11 @@
+
+## Feature updates and improvements
+
+- Add second level cache in RegistryAuthService by @munishchouhan in https://github.com/seqeralabs/wave/pull/588
```
@christopher-hakkaart, should we use code font for `RegistryAuthService`?

christopher-hakkaart replied:
Good question. I haven't been using code font in the changelog as these are mostly commit or PR messages which haven't been formatted. I think as a rule we can start to code font these to make it clearer to the reader.

christopher-hakkaart replied:
Fixed in revision

justinegeffen commented on changelog/wave/v1.10.3.mdx:
```diff
@@ -0,0 +1,11 @@
+
+## Feature updates and improvements
+
+- Added remove from redis in invalidateAuthorization by @munishchouhan in https://github.com/seqeralabs/wave/pull/591
```
Is "Remove from redis" an action or a config? Different fonts might make this clearer.

christopher-hakkaart replied:
Something in between. I've updated accordingly.

justinegeffen commented on changelog/wave/v1.10.4.mdx:
```diff
@@ -0,0 +1,13 @@
+
+## Feature updates and improvements
+
+- Remove contentreader and contentreaderfactory classes by @munishchouhan in https://github.com/seqeralabs/wave/pull/584
```
Same question about code font (and applies to all files).

christopher-hakkaart replied:
Fixed in commit

justinegeffen commented on changelog/wave/v1.11.0.mdx:
```diff
@@ -0,0 +1,15 @@
+
+## Feature updates and improvements
+
+- Add /v1alpha2/container/containerId by @munishchouhan in https://github.com/seqeralabs/wave/pull/609
```
```suggestion
- Add `/v1alpha2/container/containerId`by @munishchouhan in https://github.com/seqeralabs/wave/pull/609
```

justinegeffen commented:
This is awesome work, Chris! 

I have added two comments and one suggestion which can be applied across all the files if you agree.

One other suggestion I have is to separate the files into **Feature updates and improvements** and **Bug fixes** to align with our changelog format.

christopher-hakkaart commented on changelog/wave/v1.10.2.mdx:
```diff
@@ -0,0 +1,11 @@
+
+## Feature updates and improvements
+
+- Add second level cache in RegistryAuthService by @munishchouhan in https://github.com/seqeralabs/wave/pull/588
```
```suggestion
- Add second level cache in `RegistryAuthService` by @munishchouhan in https://github.com/seqeralabs/wave/pull/588
```

christopher-hakkaart commented on changelog/wave/v1.10.4.mdx:
```diff
@@ -0,0 +1,13 @@
+
+## Feature updates and improvements
+
+- Remove contentreader and contentreaderfactory classes by @munishchouhan in https://github.com/seqeralabs/wave/pull/584
```
```suggestion
- Remove `contentreader` and `contentreaderfactory` classes by @munishchouhan in https://github.com/seqeralabs/wave/pull/584
```

christopher-hakkaart commented on changelog/wave/v1.10.3.mdx:
```diff
@@ -0,0 +1,11 @@
+
+## Feature updates and improvements
+
+- Added remove from redis in invalidateAuthorization by @munishchouhan in https://github.com/seqeralabs/wave/pull/591
```
```suggestion
- Added remove from Redis in `invalidateAuthorization` by @munishchouhan in https://github.com/seqeralabs/wave/pull/591
```

christopher-hakkaart commented:
Thanks @justinegeffen, I've adapted the PR to incorporate your suggestions.

- I've added inline code where appropriate. For potentially ambiguous messages where something could have been a tool/package name or code, I treated them as a name. 
- I added a "Bug fixes" section for all commits and PRs described as " fixes " and renamed any pages that were only fixes.

justinegeffen commented:
@ewels, if you're happy with this it'll be good to push it through. :D

ewels reviewed (approved):
Awesome! 🙌🏻

ewels commented:
Just needs those open comments to be resolved, then it can be merged.

christopher-hakkaart commented:
Comments are resolved.
